README.md

```diff
@@ -1,5 +1,18 @@
-# license-list-XML
-This is a workspace for developing and implementing a new XML source format for the SPDX License List. This source format is (or will be) used to generate the authoritative file formats for licenses on the SPDX License List. These authoritative [target/destination formats](https://github.com/spdx/license-list-data) will remain the same for the time being.
+# SPDX License List 
+The [SPDX License List](https://spdx.org/licenses/) is a list of commonly found licenses and exceptions used in free and open source and other collaborative software or documentation. The purpose of the SPDX License List is to enable easy and efficient identification of such licenses and exceptions in an SPDX document, in source files or elsewhere. The SPDX License List includes a standardized short identifier, full name, vetted license text including matching guidelines markup as appropriate, and a canonical permanent URL for each license and exception. 
+
+# This Repository
+This repository contains the XML source and schema files used to generate the authoritative, supported SPDX list file formats, including the web pages you see at [spdx.org/licenses](https://spdx.org/licenses/) and other generated data formats found in the [SPDX license-list-data repository]. 
```

Comment:
`[SPDX license-list-data repository]` is not a link.  I think you meant:

```markdown
... the [SPDX license-list-data repository][license-list-data].
```

with a reference footer at the bottom of the file:

```markdown
[license-list-data]: https://github.com/spdx/license-list-data
```